<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Import_Mesh"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_camera_reset"/>
    <addaction name="actionTrim"/>
    <addaction name="actionScreenshot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_GotoStep0"/>
   <addaction name="separator"/>
   <addaction name="action_simulation_start"/>
   <addaction name="action_simulation_single_step"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Plots"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_camera_reset">
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_simulation_start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/media-playback-start.svg</normaloff>
     <normalon>:/gui/icons/media-playback-pause.svg</normalon>:/gui/icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/but-quit.png</normaloff>:/gui/icons/but-quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_simulation_single_step">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/system-log-out.svg</normaloff>:/gui/icons/system-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Single Step</string>
   </property>
   <property name="toolTip">
    <string>Single step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_GotoStep0">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/go-first.svg</normaloff>:/gui/icons/go-first.svg</iconset>
   </property>
   <property name="text">
    <string>GotoStep0</string>
   </property>
  </action>
  <action name="action_Import_Mesh">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/document-export.svg</normaloff>:/gui/icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>Import Mesh...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionShow_Plots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/graph.svg</normaloff>:/gui/icons/graph.svg</iconset>
   </property>
   <property name="text">
    <string>Show Plots</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/open-file-folder-svgrepo-com.svg</normaloff>:/gui/icons/open-file-folder-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/save-svgrepo-com.svg</normaloff>:/gui/icons/save-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTrim">
   <property name="text">
    <string>Trim</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/gui/icons/screenshot-svgrepo-com.svg</normaloff>:/gui/icons/screenshot-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
